Sources/Tropos/Extensions/INInteraction.swift

```diff
@@ -9,7 +9,6 @@ extension INInteraction {
         return INInteraction(intent: intent, response: nil)
     }
 
-    private static let checkWeatherSuggestedPhrase = NSLocalizedString(
-        "Check the weather",
-        comment: "Siri suggestion phrase to check weather")
+    private static let checkWeatherSuggestedPhrase = NSString
+        .deferredLocalizedIntentsString(with: "CheckWeatherSuggestedPhrase") as String
```

Comment:
This new concept of deferred localisation enables a donated intent to be localised potentially much later, when the device could have changed language.